schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "datatype": "string",
+            "array_of": true
+          }
+        ]
+      },
+      {
+        "name": "type_hgvs_expression",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "nucleotide_expression",
+            "datatype": "string"
+          },
+          {
+            "name": "protein_expression",
+            "datatype": "string"
+          },
+          {
+            "name": "molecular_consequence",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          },
+          {
+            "name": "assembly",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_nucleotide_sequence_expression",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "expression",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "sequence_type",
+            "datatype": "string"
+          },
+          {
+            "name": "sequence_accession_version",
+            "datatype": "string"
+          },
+          {
+            "name": "sequence_accession",
+            "datatype": "string"
+          },
+          {
+            "name": "sequence_version",
+            "datatype": "string"
+          },
+          {
+            "name": "change",
+            "datatype": "string"
+          },
+          {
+            "name": "assembly",
+            "datatype": "string"
+          },
+          {
+            "name": "submitted",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_protein_sequence_expression",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "expression",
+            "datatype": "string",
+            "array_of": true
+          },
+          {
+            "name": "sequence_type",
+            "datatype": "string"
+          },
+          {
+            "name": "sequence_accession_version",
+            "datatype": "string"
+          },
+          {
+            "name": "sequence_accession",
+            "datatype": "string"
+          },
+          {
+            "name": "sequence_version",
+            "datatype": "string"
+          },
+          {
+            "name": "change",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_hgvs",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "hgvs",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_aggregated_interpretation_set",
```

Comment:
Could we get rid of this table in the same way I suggest for `gene_list` and `hgvs_list`?